docs/reference/eql/eql.asciidoc

```diff
@@ -136,6 +136,49 @@ GET /my-index-000001/_eql/search
 ----
 // TEST[setup:sec_logs]
 
+Use the <<common-options-response-filtering,`filter_path`>> query parameter to
+filter the properties returned in the API response. For example, the following
```

Comment:
I wouldn't call those elements "properties". I think it's more like components or elements. Or maybe leave out a name for them and just state that it's "filtering the response content".

Reply:
Thanks @astefan! I've reworded to avoid `properties`.